resource_calendar_rrule/models/resource_calendar_attendance.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import datetime
+import pytz
+from dateutil import rrule
+from openerp import api, fields, models
+try:
+    from openerp.addons.field_rrule import FieldRRule
+except ImportError:
+    FieldRRule = fields.Serialized
+
+
+class ResourceCalendarAttendance(models.Model):
+    _inherit = 'resource.calendar.attendance'
+
+    rrule = FieldRRule(
+        'Repetition', help='Add an arbitrary amount of repetition rules.',
+        required=True, default=lambda self: self._default_rrule(),
+    )
+    rrule_preview = fields.Text('Preview', compute='_compute_rrule_preview')
+
+    @api.model
+    def create(self, vals):
+        # if some legacy code creates an attendance, set up a repetition rule
+        # for it
+        if 'rrule' not in vals and 'dayofweek' in vals:
+            rrule = self._default_rrule()
+            rrule[0]['byweekday'] = [vals['dayofweek']]
+            vals['rrule'] = rrule
+        return super(ResourceCalendarAttendance, self).create(vals)
+
+    @api.multi
+    def _compute_rrule_preview(self):
+        language = self.env['res.lang'].search([
+            ('code', '=', self.env.context.get('lang', self.env.user.lang))
+        ])
+        for this in self:
+            text = ''
+            for start, end in this._iter_rrule(
+                    datetime.datetime.now(),
+                    datetime.datetime.now() + datetime.timedelta(days=7),
+            ):
+                text += '%s %s - %s\n' % (
+                    fields.Datetime.context_timestamp(this, start)
+                    .strftime(language.date_format),
+                    fields.Datetime.context_timestamp(this, start)
+                    .strftime(language.time_format),
```

Comment:
@hbrunn is it posible to use strftime("%H:%M") instead of language.time_format? The preview field always show different second depend on current time. It may confuse end user.

![rrule1](https://cloud.githubusercontent.com/assets/1947376/23285050/88942218-fa61-11e6-9513-4c42d63d065b.png)

![rrule2](https://cloud.githubusercontent.com/assets/1947376/23285058/8fbeb95e-fa61-11e6-9fa2-8bd2e3ca91b4.png)